source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -796,8 +945,16 @@ SplitRequestPtr PubSubRequest::create(Router& router, Common::Redis::RespValuePt
       new PubSubRequest(callbacks, command_stats, time_source, delay_command_latency)};
   std::string key = std::string();
   int32_t redisShardsCount=0;
+  int32_t requestsCount=1;
   int32_t shard_index=0;
-  
+  bool iserror = false;
+  std::string firstarg = std::string();
+
+  if (incoming_request->asArray().size() > 1) {
+    firstarg = absl::AsciiStrToLower(incoming_request->asArray()[1].asString());
```

Comment:
why is only first arg taken , the subscribe command can accept , n number of channels , and the _keyspace_ can be nth channel